File: PooCC01CNetz/Semana2/Python/Ejercicios3/AgenciaSM.py
from Automovil import Automovil

class AgenciaSM:
    def main(self):
        # Crear un objeto de tipo automovil
        a1 = Automovil()
        a1.modelo = "Prius"
        a1.precio = 350000
        a1.anio = 2022
        print("Modelo      :", a1.modelo)
        print("Precio      :", a1.precio)
        print("Anio        :", a1.anio)
        res = a1.verifica(2023)
        print("Verifica    :", res, "\n")

        a2 = Automovil("Chevy", 70000)
        print("Modelo      :", a2.modelo)
        print("Precio      :", a2.precio)
        print("Anio        :", a2.anio)
        res2 = a2.verifica(2023)
        print("Verifica    :", res2)

if __name__ == "__main__":
    agencia = AgenciaSM()
    agencia.main()

File: PooCC01CNetz/Semana02/Java/Ejercicios3/AgenciaSM.java
public class AgenciaSM {
    public static void main(String[] args) {
        // Crear un objeto de tipo automovil
        Automovil a1 = new Automovil();

        // Inicializar datos del objeto
        a1.modelo = "Prius";
        a1.precio = 350000;
        a1.anio = 2022;

        // Impresion
        System.out.println("Modelo      : " + a1.modelo);
        System.out.println("Precio      : " + a1.precio);
        System.out.println("Anio        : " + a1.anio);
        boolean res = a1.verifica(2023);
        System.out.println("Verifica    : " + res + "\n");

        Automovil a2 = new Automovil("Chevy", 70000);
        System.out.println("Modelo      : " + a2.modelo);
        System.out.println("Precio      : " + a2.precio);
        System.out.println("Anio        : " + a2.anio);
        boolean res2 = a2.verifica(2023);
        System.out.println("Verifica    : " + res2);
    }
}
